geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseFeatureReader.scala

```diff
@@ -40,18 +42,23 @@ class HBaseFeatureReader(table: Table,
 
   val scanners = scans.map(table.getScanner)
 
-  val results =
-    try {
-      scanners
-          .flatMap(_.toList)
-          .flatMap { r => r.getFamilyMap(HBaseDataStore.DATA_FAMILY_NAME).values().map(serde.deserialize) }
-          .iterator
-    } catch {
-      case e: Exception =>
-        logger.error("Error creating scan", e)
-        closeScanners()
-        Iterator.empty
-    }
+  val unfilterResults =  try {
+    scanners
+      .flatMap(_.toList)
+      .flatMap { r => r.getFamilyMap(HBaseDataStore.DATA_FAMILY_NAME).values().map(serde.deserialize) }
+      .iterator
+  } catch {
+    case e: Exception =>
+      logger.error("Error creating scan", e)
+      closeScanners()
```

Comment:
i realize it was like this before, but do we need to close scanners if there isn't an exception? probably would have to have a closeableIterator